src/sentry/api/bases/organization_events.py

```diff
@@ -46,7 +46,7 @@ def get_snuba_params(self, request, organization, check_global_views=True):
 
             params = self.get_filter_params(request, organization)
             params = self.quantize_date_params(request, params)
-            params["user_id"] = request.user.id
+            params["user_id"] = request.user.id if request.user else None
```

Comment:
We don't have any existing tests for making requests via API tokens should we have one?

Reply:
Yes. It'll be nice if there is a test

Reply:
Yeah seems reasonable